.github/workflows/main.yml

```
@@ -155,6 +155,11 @@ jobs:
               --api server-server \
               -r "$RELEASE" \
               -o spec/server-server-api/api.json
+          scripts/dump-openapi.py \
+              --base-url "https://spec.matrix.org${{ needs.calculate-baseurl.outputs.baseURL }}" \
+              --api identity \
+              -r "$RELEASE" \
+              -o spec/identity-api/api.json
```

This should match the path of the website:


```suggestion
              -o spec/identity-service-api/api.json
```

---

applied